### ISSUE TITLE
hasFinitePrecision removed from api

### ISSUE THREAD

**Issue by bsutton** [a14n/dart-decimal#83]
I'm trying to update the Fixed package to the latest version of decimal (2.2.0)

The fixed package currently uses Decimal.hasFinitePrecision

```dart
  /// Creates a fixed scale decimal from [amount] with
  /// the given [scale].
  ///
  /// [scale] defaults to 16 if not passed.
  Fixed.fromDecimal(Decimal amount, {this.scale = 16}) {
    _checkScale(scale);
    minorUnits = _rescale(
      (amount * Decimal.ten.pow(amount.scale)).toBigInt(),
      existingScale: amount.hasFinitePrecision ? amount.scale : 16,
      targetScale: scale,
    );
  }
```

The removal isn't identified in the breaking changes and the 2.0 change log mentions it being added to Rational (which does actually have the method either).

Is there an alternate way of determine if we have finite precisions or are we now saying that decimals always have finite precision?

**Comment by a14n** [comment]
> Is there an alternate way of determine if we have finite precisions or are we now saying that decimals always have finite precision?

Yes, starting from 2.0 a `Decimal` always has finite precision. I will send you a PR for the update of fixed package.

**Comment by a14n** [comment]
> I will send you a PR for the update of fixed package.

Looks like you already made the migration. Do you still have problem?
